lib/gui/components/update-notifier/templates/update-notifier-modal.tpl.html

```diff
@@ -6,6 +6,9 @@ <h4 class="modal-title">New Update Available!</h4>
 <div class="modal-body">
   <div class="modal-text">
     <p>A new version of Etcher is available for download</p>
+    <a os-open-external="https://github.com/resin-io/etcher/blob/master/CHANGELOG.md#readme">
```

Comment:
<img width="523" alt="screenshot 2016-12-01 12 28 07" src="https://cloud.githubusercontent.com/assets/2192773/20802194/a3ba8d76-b7c1-11e6-91f4-8eb52221e4a2.png">

Indeed, very interesting. Looks good to me then!